src/main/java/stream/flarebot/flarebot/util/GeneralUtils.java

```diff
@@ -193,4 +212,38 @@ public static Role getRole(String s, String guildId) {
         }
         return null;
     }
+
+    public static Response post(String url, MediaType type, String body) throws IOException {
+        Request.Builder request = new Request.Builder().url("https://hastebin.com/documents");
+        RequestBody requestBody = RequestBody.create(type, body);
+        request = request.post(requestBody);
+        return FlareBot.getOkHttpClient().newCall(request.build()).execute();
```

Comment:
Should call the other post rather than also doing this. You know how method hierarchy works